app/src/main/java/com/practicum/playlistmaker/media/presentation/viewmodel/FavoriteViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.practicum.playlistmaker.media.presentation.viewmodel
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.practicum.playlistmaker.R
+import com.practicum.playlistmaker.media.presentation.state.FavoriteState
+
+class FavoriteViewModel(
+    application: Application
+) : AndroidViewModel(application) {
+
+    private var screenStateLiveData = MutableLiveData<FavoriteState>(FavoriteState.EmptyContent(application.getString(R.string.favorites_empty)))
```

Comment:
Так как сама ссылка не меняется на стейт после создания вьюмодели, то можно сделать так:
```suggestion
    private val screenStateLiveData = MutableLiveData<FavoriteState>(FavoriteState.EmptyContent(application.getString(R.string.favorites_empty)))
```